node/pkg/watchers/evm/watcher.go

```diff
@@ -661,7 +592,7 @@ func (w *Watcher) Run(parentCtx context.Context) error {
 
 				start := time.Now()
 				currentHash := ev.Hash
-				logger.Debug("processing new header",
+				logger.Info("processing new header",
```

Comment:
Do we want to keep this as Info? feel like that will cause some log spam

Reply:
Doh! Didn't mean to commit that. Changed it back.